tests/test_image.py

```diff
@@ -0,0 +1,53 @@
+from morphocut import Pipeline
+from morphocut.image import ThresholdConst, Rescale, ImageWriter
+
+import numpy as np
+import skimage.io
+
+import pytest
+
+def test_ThresholdConst():
+    image = skimage.data.camera()
+    with Pipeline() as pipeline:
+        result = ThresholdConst(image, 256)
+
+    stream = pipeline.transform_stream()
+    pipeline.run()
+
+def test_Rescale():
+    image = skimage.data.camera()
+    with Pipeline() as pipeline:
+        result = Rescale(image, in_range=(0, 200), dtype=np.uint8)
+
+    stream = pipeline.transform_stream()
+    pipeline.run()
+
+#TODO: Confused in fmt and meta. 
+@pytest.mark.xfail(strict=True)
+def test_ImageWriter():
+    image_root = "/tests/images/"
+    fmt = "{ext}"
+    meta = ...
+    image = skimage.data.camera()
+    with Pipeline() as pipeline:
+        result = ImageWriter(image_root, fmt, image, meta)
+
+    stream = pipeline.transform_stream()
+    pipeline.run()
+
+#transform_stream method of FindRegion has an error because of which test fails
+#Also ExtractROI class is dependent on FindRegion so couldn't write test of it either
+
+
+'''
+@pytest.mark.xfail(strict=True)
+def test_FindRegions():
+    image = skimage.data.camera()
+    mask = ThresholdConst(image, 255)
```

Comment:
Has to be called in a pipeline context.

Reply:
I tried that too. but it is actually giving error in image.py file inside 'FindRegions' class. It says that 

> module 'skimage' has no attribute 'measure'

Reply:
I don't have this error but it is obvious that a `import skimage.measure` is missing in `image.py`.